### PR TITLE
GH-999 Bumps AmCharts Version to 3.17.0

### PR DESCRIPTION
**Pull request by mlzummo**
Changes AmCharts bower and github repo to reflect official channel

Fixes GH-999

Signed-off-by: Mathew Zummo <mzummo@gmail.com>

### CODE REVIEW

**Comment by GordonSmith** [comment]
Needs rebase.

File: bower.json [hunk 1]
```diff
@@ -41,8 +41,9 @@
     "font-awesome": "~4.3.0",
     "topojson": "~1.6.19",
     "dagre": "https://raw.githubusercontent.com/cpettitt/dagre/v0.7.1/dist/dagre.js",
+    "d3-cloud": "^1.1.2",
```
Comment (GordonSmith) [inline bower.json]: should not be here

**Comment by mlzummo** [comment]
@GordonSmith Please Pull

File: bower.json [hunk 2]
```diff
@@ -42,7 +42,8 @@
     "topojson": "~1.6.19",
     "dagre": "https://raw.githubusercontent.com/cpettitt/dagre/v0.7.1/dist/dagre.js",
     "d3-cloud": "~1.1.2",
-    "amcharts": "~3.14.1",
+    "amcharts3": "~3.17.0",
+    "d3-cloud": "~1.1.2",
```
Comment (GordonSmith) [inline bower.json]: Remove additional d3-cloud reference

**Pull request merged**